bokeh/core/json_encoder.py

```diff
@@ -61,8 +61,14 @@
 # Bokeh imports
 from ..settings import settings
 from ..util.dependencies import import_optional
-from ..util.serialization import (convert_datetime_type, convert_timedelta_type, is_datetime_type,
-                                  is_timedelta_type, transform_array, transform_series)
+from ..util.serialization import (
+    convert_datetime_type,
+    convert_timedelta_type,
+    is_datetime_type,
+    is_timedelta_type,
+    transform_array,
+    transform_series
```

Comment:
Definitively better than the original. Seems wasteful on the vertical space, but I suppose will aid in diff quality. However, to have the diff argument fully hold, we will need to use trailing commas. This can be done separately, as we should be enforcing trailing commas more generally, as we do in TypeScript.

Reply:
Here is a black-compatible sort config, I will update later:
```
[settings]
multi_line_output=3
include_trailing_comma=True
force_grid_wrap=0
use_parentheses=True
line_length=88
```